src/main.h

```diff
@@ -461,7 +461,8 @@ bool ReadBlockFromDisk(CBlock& block, const CBlockIndex* pindex);
 bool DisconnectBlock(CBlock& block, CValidationState& state, CBlockIndex* pindex, CCoinsViewCache& coins, bool* pfClean = NULL);
 
 /** Reprocess a number of blocks to try and get on the correct chain again **/
-bool DisconnectBlocksAndReprocess(int blocks);
+bool DisconnectBlocks(int blocks);
+void ReprocessBlocks(int nBlocks);
```

Comment:
for code readability; these should at least be consistent.  ReprocessBlocks(nBlocks) passes nBlocks to DisconnectBlocks(blocks).  Ideally DisconnectBlocks should be changed from `blocks` to `nBlocks`.